app/src/main/java/com/gaurav/avnc/ui/vnc/VirtualKeys.kt

```diff
@@ -48,6 +48,13 @@ class VirtualKeys(activity: VncActivity) {
         }
     }
 
+    fun releaseMetaKeys() {
+        val binding = stub.binding as VirtualKeysBinding
+        binding.shiftBtn.toggle()
+        binding.ctrlBtn.toggle()
+        binding.altBtn.toggle()
```

Comment:
Please note that `stub.binding` will be `null` until virtual keys are inflated, so we have to use safe cast `as?` here.
Also,  `toggle()` will *invert* the button state, but we want to always release the buttons.